src/main/webapp/js/translate-aboutus-loader.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
It seems like this code is very similar between the two loader.js and and the inline javascript in translate.html. Perhaps you could make the requestTranslation() function more generic and use it throughout.

Reply:
Yes the two files are similar. Translate-loader is for a separate page which I think we may not need it in the future so I separate the two files.